spec/formatters/ltsv_spec.rb

```diff
@@ -24,4 +24,8 @@
   it 'escapes escape sequences as value' do
     expect(subject).to include('will_escaped:\\t')
   end
+
+  it 'should be splitted by tab' do
```

Comment:
Can we change this to more definitive language, perhaps `it 'is splitted by hard tabs' do`?